sdcm/db_stats.py

```diff
@@ -413,6 +430,42 @@ def calculate_stats_average(self):
         if total_stats:
             self._stats['results']['stats_total'] = total_stats
 
+    def _cut_comments(self, result):
+        return '\n'.join([i.strip() for i in result.split('\n') if not i.startswith('#')])
+
+    def build_archive(self):
+        download_files = ['/proc/meminfo', '/proc/cpuinfo', '/proc/interrupts', '/proc/vmstat']
+        storing_dir = os.path.join(self.db_cluster.logdir, 'tmp')
```

Comment:
better to  call directory `system_data` instead of tmp

Reply:
done